libr/core/cbin.c

```diff
@@ -2491,6 +2491,41 @@ struct io_bin_section_info_t {
 	int fd;
 };
 
+/* Map Sections to Segments https://github.com/radareorg/radare2/issues/14647 */
+static int rabin_map_sections_to_segments(RBin *bin) {
+	RListIter *iter, *iter2;
+	RBinSection *section = NULL, *segment = NULL;
+	RList *sections = r_list_new();
+	RList *segments = r_list_new();
+	RList *tmp = r_bin_get_sections (bin);
+
+	printf ("Section to Segment mapping:\n");
```

Comment:
I believe @radare suggested to use RTable api, right? I'm not sure how that would work in this case, have you tried it?

Reply:
It uses RTable API now, thanks for the suggestion.

Reply:
use r_cons_printf for now